src/SplitProvider.tsx

```diff
@@ -89,10 +93,16 @@ const SplitProvider = ({
      * Instantiating a new factory
      * @link https://help.split.io/hc/en-us/articles/360020448791-JavaScript-SDK
      */
-    const factory: SplitIO.ISDK = SplitFactory(sdkConfig);
-    const nextClient = factory.client(config.core.key, config.core.trafficType);
+    const nextFactory: SplitIO.ISDK = SplitFactory(sdkConfig);
+    const nextClient = nextFactory.client(
```

Comment:
Since this is the default client we don't need to pass `key` neither `trafficType`.

Reply:
Either way it is the same client instance returned. I can remove though.